README.md

```diff
@@ -14,8 +30,9 @@ NOTION_BASE_URL = '[YOUR NOTION BASE URL]' # for example https://www.notion.so/u
 # optional:
 CONFIG = {
   'db_id' => '[ID OF THE NOTION DATABASE YOU WANT TO ACCESS]',
-  'chosen_filter_property_name' => '[TO FILTER, PUT PROPERTY NAME HERE]', # e.g. 'Status'
-  'filter_options' => ['In Progress', 'Priority'] # <-- e.g.
+  'filter_property_name' => '[TO FILTER, PUT PROPERTY NAME HERE]', # e.g. what you've named the column in Notion, e.g. 'Status'
```

Comment:
`notionpaper.rb` gets the filter property by getting `config['chosen_filter_property_name']`, not `config['filter_property_name']`. Did you want to change the name of the key everywhere?

(You also use 'filter_property_name` on line 70)